lib/tests/behat/collapsable_section_output.feature

```diff
@@ -0,0 +1,66 @@
+@core @javascript
+Feature: Test collapsable section output module
+  In order to show extra information to the user
+  As a user
+  I need to interact with the collapsable section output
+
+  Background:
+    # Get to the fixture page.
+    Given I log in as "admin"
+    And I am on fixture page "/lib/tests/behat/fixtures/collapsable_section_output_testpage.php"
+
+  Scenario: Collapsable sections can be opened and closed
+    Given I should not see "Dialog content"
```

Comment:
This step is moved from background to the first scenario to test it only ones.

Reply:
Thanks for changing it. I know it's a little thing, but improves the performance a bit :-)